Automattic/simplenote-ios
# Enhanced Sort Options

### Details:
In this PR we're matching Android's and Simplenote.com's Sort Options for the notes list.

Closes #204

### Testing:
1. Launch Simplenote on your iOS device
 1.1. Press over the top left icon
 1.2. Press over **Settings**
 1.3. Press over the first row: `Sort Order`
2. Log into Simplenote.com (same account as in step 1)
 2.1. Press over your account's email in the top right corner
 2.2. Press over **Settings**  
 2.3. Press over the **Edit** button in Display > Sorting
3. Modify the Sort Order in both iOS and Web
4. Compare both apps, and verify the notes are displayed in the same order

cc @bummytime
Thanks in advance sir!
